pods-beaver-themer.php

```diff
@@ -398,21 +404,77 @@ function pods_beaver_render_settings_field_order_by( $field, $name, $settings )
 /**
  * Work around UABB overly aggressive setting of post_type
  *
- * @param array  $args
+ * @param array $args
  * @param object $settings
  *
  * @return array
  *
  * @since 1.1
  */
-function pods_beaver_uabb_blog_posts( $args, $settings ) {
-
-	if ( empty( $settings->use_pods ) || 'no' === $settings->use_pods ) {
-		return $args;
-	}
+function pods_beaver_uabb_blog_posts( $args ) {
 
 	$args['post_type'] = 'any';
+	remove_filter( 'fl_builder_loop_query_args', 'pods_beaver_uabb_blog_posts' );
 
 	return $args;
+}
+
+/**
+ * Adds PODS relation as data source for posts module.
+ *
+ * @since 1.3
+ *
+ * @param array $field
+ * @param string $name The field name.
+ * @param object $settings
+ *
+ * @return array
+ */
+function pods_beaver_render_settings_field( $field, $name, $settings ) {
+	if ( 'data_source' != $name ) {
+		return $field;
+	}
 
+	$field['options']['pods_relationship'] = __( 'PODS Relationship', 'pods-beaver-builder-themer-add-on' );
```

Comment:
Can you fix the caps on PODS to be Pods?

Reply:
yes will go over all labels before the Release